pages/use-case/software-and-technology.vue

```diff
@@ -68,18 +67,19 @@ import CallToActionSection from '@/components/CallToActionSection.vue'
 import FAQwithCategories from '../../components/FAQwithCategories.vue'
 import softwareAndTechnologyFaqs from '../../faqs/use-case/software-and-technology-faqs'
 import getSiteMeta from '@/utils/getSiteMeta'
+import MostUsedFeatures from '../../components/home/MostUsedFeatures.vue'
```

Comment:
```suggestion
import MostUsedFeatures from '@/components/home/MostUsedFeatures.vue'
```
I know that we have previously imported components like this without using @ import.
It will be better if we can use @ import.